src/pages/index.tsx

```diff
@@ -1,5 +1,38 @@
 import CampaignHero from "../components/hero/campaignHero";
+import {
+  SkipNavigation,
+  Template,
+  TemplateAboveHeader,
+  TemplateBreakout,
+  TemplateContent,
+  TemplateFooter,
+  TemplateHeader,
+} from "@nypl/design-system-react-components";
 
 export default function Home() {
-  return <CampaignHero />;
+  return (
+    <>
+      <SkipNavigation />
+      <Template>
```

Comment:
Just curious, did you try the `TemplateAppContainer` component as well? You don't need to change it but just wondering if there were any issues with it.

Reply:
Refactored to use `TemplateAppContainer`